### PR TITLE
Add icons

### PR DESCRIPTION
**Pull request by kiyaGu**
## Why?

We conducted a user testing for annotating relationships and this PR addresses some of the feedback we got

## What?

-   add tooltip and info icon for multiple choice options

### nested
Screenshot 2020-03-12 at 16 38 56

### not nested
Screenshot 2020-03-13 at 09 54 21

### CODE REVIEW

File: packages/tc-ui/src/primitives/multiple-choice/server.jsx [hunk 1]
```diff
@@ -13,6 +14,42 @@ const localOnChange = (event, currentValues, onChange) => {
...
+			className="tooltip-container"
+			id={`tooltip-target-${parentCode}-${checkboxValue}`}
+		>
+			<span id={`tooltip-target-${parentCode}-${checkboxValue}`} />
+			<i
+				aria-label={`help for ${checkboxValue}`}
+				className="o-icons-icon o-icons-icon--info treecreeper-help-icon"
+			/>
+			<div
+				className={getClass(isNested)}
```
Comment (wheresrhys) [inline packages/tc-ui/src/primitives/multiple-choice/server.jsx]: I think we should use this when not nested too (we don't have any examples elsewhere in Biz Ops, but I think it's a nice feature)

Comment (wheresrhys) [reply packages/tc-ui/src/primitives/multiple-choice/server.jsx]: ...but I see in the css that implementing that would be extra work, so it's fine to leave it as it is for now

Comment (kiyaGu) [reply packages/tc-ui/src/primitives/multiple-choice/server.jsx]: @wheresrhys I have modified it to address your suggestion

Comment (wheresrhys) [reply packages/tc-ui/src/primitives/multiple-choice/server.jsx]: 🙌

**Pull request merged**